docs/fundamentals/code-analysis/quality-rules/ca1001.md

```diff
@@ -41,6 +43,57 @@ To fix a violation of this rule, implement <xref:System.IDisposable> and from th
 
 In general, do not suppress a warning from this rule. It is OK to suppress the warning when the dispose ownership of the field(s) is not held by the containing type.
 
+## Configurability
+
+This rule has the following configurable options that can be configured for just this rule, for all rules, or for all rules in this category (Design). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
+
+### Include or exclude selected APIs
```

Comment:
I have decided to keep this title even though some rules don't have the `api_surface` configuration to keep a standard look across pages, let me know if you prefer not to do so.

Reply:
I forgot to include this part!